hashing/legacyKeccak256/legacyKeccak256.go

```diff
@@ -0,0 +1,28 @@
+package legacyKeccak256
```

Comment:
As I see we already have the legacy keccak implementation under hashing/keccak/keccak.go so you can use that instead.

Reply:
done. used that